SimplenoteUITests/EmailLogin.swift

```diff
@@ -38,6 +37,7 @@ class EmailLogin {
         enterPassword(enteredValue: password)
         app.buttons[UID.Button.logIn].tap()
         handleSavePasswordPrompt()
+        waitForSpinnerToDisappear()
```

Comment:
Not really a part of the fix, and not necessary, but it does not hurt, and does not occupy much time, since the function waits for the spinner to disappear. So, if the spinner was not there in the first place, and will be completed really fast.